src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -32,6 +32,16 @@
         Upload/Edit Photo
       </button>
     </div>
+    <br>
+    <br>
+    <div class="">
+      <button id="deletePhoto"
+              class="btn btn-primary"
+              href="${profile.deleteUrl}"
```

Comment:
`href` is not semantically an accepted attribute for `button`. If you want to put a data, use [jQuery's `data-*`](https://api.jquery.com/data/) attribute.